Icon/Icons/AnalyticsIcon.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { PropTypes } from 'react';
+import Icon from '../../Icon';
+
+const AnalyticsIcon = ({ size }) =>
```

Comment:
Would probably be good to use the same api for each of these to make them customizable in the same ways.

```
const AnalyticsIcon = ({ size, color }) => {};
```